web/app/js/components/ServiceMesh.jsx

```diff
@@ -155,10 +154,9 @@ export default class ServiceMesh extends React.Component {
   }
 
   processComponents(pods) {
-    let ns = this.props.namespace + "/";
     let podIndex = _(pods)
-      .filter(p => _.startsWith(p.deployment, ns))
-      .groupBy(p => _.replace(p.deployment, ns, ""))
+      .filter(p => p.controlPlane)
+      .groupBy(p => {return p.deployment.substring(p.deployment.indexOf('/')+1)})
```

Comment:
I think if you remove the curly braces here, then you won't need the explicit return. Could also use some lodash helpers. Does this work:

```
      .groupBy(p => _.last(_.split(p.deployment, "/")))
```

Reply:
I was thinking of submitting a separate PR that took care of removing the `-namespace` flag in the Go code since, this PR was just frontend related. But I can include that flag change with this PR.